### DISCUSSION TITLE
How to config the show the signed status of commits

### DISCUSSION THREAD

**Discussion by LintaoAmons** [rbong/vim-flog#117] [Q&A]
like `--show-signature`, show the signed information of the commits

**Comment by rbong** [comment]
Since multiline commit messages are supported and you can send raw arguments to Flog, you can use this option as a one-off this way:

```
:Flog -- --show-signature
```

Or, you can set it to a default option in your vimrc:

```
let g:flog_permanent_default_opts = { 'raw_args': '--show-signature' }
```

Though using the way above, if you pass any raw args to Flog, it will be overwritten.

There will also be no highlighting for the signature, which would be ideal but we don't have.

I would accept a PR for this feature, as long as it came with a warning in the documentation that Flog can't tell the difference between real GPG keys and fake GPG keys embedded in commit messages, and that the safest way to check them is through the Git CLI.
